__tests__/app.test.js

```diff
@@ -98,22 +98,45 @@ describe("GET: /api/articles", () => {
         expect(articles).toBeSortedBy("topic", { ascending: true });
       });
   });
-  test("400: responds with an appropriate error message when given an invalid sort_by query", () => {
+  test("200: responds with articles filtered by topic", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              topic: "mitch",
+            })
+          );
+        });
+      });
+  });
+  test("404: responds with an appropriate error message when given an invalid sort_by query", () => {
     return request(app)
       .get("/api/articles?sort_by=invalid")
-      .expect(400)
+      .expect(404)
```

Comment:
We'd still want a 400 error code here, as an invalid sort by query counts as a bad request